neo/rawio/openephysrawio.py

```diff
@@ -115,9 +116,11 @@ def _parse_header(self):
 
                 # check for continuity (no gaps)
                 diff = np.diff(data_chan['timestamp'])
```

Comment:
We have to keep in mind that this diff have a very high cost when opening.
on spiekinterface side this means when parallel processing making n_jobs time this check!!!

Reply:
@samuelgarcia Feel free to make a proposal on this in a separate PR.